exporter/opentelemetry-exporter-otlp-proto-grpc/tests/metrics/test_otlp_metrics_exporter.py

```diff
@@ -0,0 +1,70 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
This file is named `test_....` but it does not seem to include any tests.

Reply:
Added tests.